docs/scripts_md/database_files_update.md

```diff
@@ -0,0 +1,72 @@
+# NAME
+
+database\_files\_update.pl -- Updates path stored in `files` and
+`parameter_file` tables so that they are relative to data\_dir
+
+# SYNOPSIS
+
+perl database\_files\_update.pl `[options]`
+
+Available option is:
+
+\-profile: name of the config file in ../dicom-archive/.loris\_mri
+
+# DESCRIPTION
+
+This script updates the path stored in the `files` and `parameter_file`
+tables to remove the <\\$data\_dir> part of the path for security improvements.
```

Comment:
the `$data_dir` looks weird in the .md file.

Reply:
should be fixed now.